package.json

```diff
@@ -88,6 +94,7 @@
     "husky": "9.1.7",
     "jest": "29.7.0",
     "metro-react-native-babel-preset": "0.77.0",
+    "next": "15.1.1",
```

Comment:
why do we need to depend on next?

Reply:
it's in dev dependencies for `useServerUnistyles` types